Assets/GothicVR/Scripts/World/GameTime.cs

```diff
@@ -11,8 +11,13 @@ public class GameTime : SingletonBehaviour<GameTime>
         private static DateTime MIN_TIME = new(1, 1, 1, 0, 0, 0);
         private static DateTime MAX_TIME = new(1, 1, 1, 23, 59, 59);
 
-        private DebugSettings.SunMovementPerformance sunPerformanceSetting;
+        public DateTimeGameEvent secondChangeChannel;
```

Comment:
Why don't we just use Unity's built in UnityAction event? What's the benefit of this additional abstraction/hop?

I used UnityEvents in this class as well. Worked fine:
https://github.com/GothicVRProject/GothicVR/blob/main/Assets/GothicVR/Scripts/Phoenix/Interface/Vm/VmGothicBridge.cs#L48

Reply:
Altered to UnityEvents.